src/main/java/sk/hfa/blog/services/BlogService.java

```diff
@@ -52,7 +56,8 @@ public void deleteById(Long id) {
         if (Objects.isNull(id))
             throw new IllegalArgumentException(Constants.INVALID_IDENTIFIER_MESSAGE);
 
-        imageService.deleteBlogArticleImage(id);
+        BlogArticle blogArticle = findById(id);
+        imageService.deleteImage(blogArticle.getTitleImage());
```

Comment:
to iste

Reply:
@RandomUserIK

Reply:
na toto chcem pocut aj vas nazor, poriesime na calle